plugins/aep-consent-registered/plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "category": "AEP Edge Consent",
```

Comment:
How about we rename this category to `Consent for Edge Network` to be more inline with the extension name and use the same category name for all consent validators